README.md

```diff
@@ -1,81 +1,70 @@
-## Synchronisation & Encryption
-#### Synchronisation
-The sync process consists in having two actors, one is the ClientServer, which exists on a Virtual Machine and the second one being the CentralServer.
+## Synchronisation and Encryption
 
-##### Setup ClientServer
-The fallowing script needs to be run on TAO instance in order to make a client server.
+### Synchronisation
+The synchronization process is based upon two actors, the Client Server, which is installed on a Virtual Machine, and the Central Server.
+
+#### Setting up the Client Server
+The following script needs to be run on a TAO instance in order to create a Client Server:
 
 ```php
 sudo -u www-data php index.php '\oat\taoOffline\scripts\tools\setup\SetupClientServer'
 ```
-    In case taoEncryption it's installed this script it's gonna setup the instance with encryption.
+
+On systems where [extension-tao-encryption](https://github.com/oat-sa/extension-tao-encryption) is installed, the script will set it up with encryption.
     
- In order to point this instance to a specific server the fallowing command needs to be run
+Point the instance to a specific server by executing the following command:
  
  ```php
```

Comment:
Actually these are not php code snippets, `shell script` should be used instead

Reply:
done